Answers.md

```diff
@@ -1,9 +1,24 @@
 # Answers
 
 1. What is React JS? How does it differ from other JavaScript frameworks?
-
+React JS is a Javascript Library for building user interface.
+What makes React really different is that it completely takes care of managing the DOM for you. It does this by using something called the Virtual DOM. Unlike other frameworks, you do everything in react by working with Components instead of Models, views, and controllers. Another thing that makes React unique is the One day data flow structure.
 

```

Comment:
I think another thing that really sets it apart from other JavaScript libraries is the fact that it is basically just JavaScript.  Everything that you see React doing can be easily related to things that you've reviewed with JavaScript.  An example would be how rendering components and passing props is very much like instantiating objects and passing values to constructor functions.  It's the same idea.